#ifndef MALWAREINCIDENTEDITOR_H
#define MALWAREINCIDENTEDITOR_H
#include <QSqlQuery>

#include <QDialog>

namespace Ui {
class MalwareIncidentEditor;
}

class MalwareIncidentEditor : public QDialog
{
    Q_OBJECT

public:
    explicit MalwareIncidentEditor(QWidget *parent = nullptr);
    ~MalwareIncidentEditor();
    void loadIncidentDetails(int incidentID);
    void saveIncident(int incidentID);

private slots:


    void on_pushButton_clicked();

private:
    Ui::MalwareIncidentEditor *ui;
    int incidentIDStore;
};

#endif // MALWAREINCIDENTEDITOR_H
